### ISSUE TITLE
Post un-minified source code so that other people can help with development!!  :)

### ISSUE THREAD

**Issue by Gkleinereva** [denehyg/reveal.js-menu#70]
Hey Greg @denehyg -

I notice that this repository doesn't have the source files for the plugin, and only includes the distribution needed to use the plugin as is.  A colleague and I are considering a few enhancements (either in a fork of the repo, or hopefully in this repo via pull request), and would love to get our hands on the un-minified source so that we can further explore that option.

Is there any chance you can add the raw source code to the repository?  How have other people contributed to the project?  If there's a way we can explore making our enhancements without you posting the raw source, that would be awesome too.  We just don't have a good way right now to move forward on contributing to the plugin with what is currently available.  

Thanks!
Evan

**Comment by denehyg** [comment]
The source is the plugin.js file. This is packaged into the menu.js file via the gulp build task. I’m happy to get pull requests but note my time on this project is limited at the moment

**Comment by Gkleinereva** [comment]
Hey @denehyg, I'm trying to build your original minified plugin files with gulp and I'm getting an error that I can't seem to beat.  I haven't modified either plugin.js or the gulp file, but it produces a different output than your minified files resulting in a console error and Reveal failing to initialize.  If you have a minute, can you point me in the right direction?  I've included the complete files below, but here are a few snippets:

### First bit of your (working) menu.js

!function(e,t){"object"==typeof exports&&"undefined"!=typeof module?module.exports=t():"function"==typeof define&&define.amd?define(t):(e=e||self).RevealMenu=t()}(this,(function(){"use strict";var e="undefined"!=typeof globalThis?globalThis:"undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{};function t(e,t,n){return e(n={path:t,exports:{},require:function(e,t){return function(){throw new Error("Dynamic requires are not currently supported by @rollup/plugin-commonjs")}

### First bit of my gulped menu.js (generated with only the command 'gulp' with all dependencies installed exactly as specified in package.json with master's plugin.js)

!function(t,e){"object"==typeof exports&&"undefined"!=typeof module?module.exports=e():"function"==typeof define&&define.amd?define(e):(t=t||self).RevealMenu=e()}(this,(function(){"use strict";function t(e){return(t="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(t){return typeof t}:function(t){return t&&"function"==typeof Symbol&&t.constructor===Symbol&&t!==Symbol.prototype?"symbol":typeof t})(e)}var e="undefined"!=typeof globalThis?globalThis:"undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{};


My test presentation works with master's menu.js, but produces the error "TypeError: xo is not a function" with my compiled menu.js.  I've spent a few hours looking through gulp docs and verifying that my setup is correct.  I've never used gulp before trying to work with this plugin.  Sorry to bother you, but any help you can offer would be greatly appreciated!  :)
[Example menu files.zip](https://github.com/denehyg/reveal.js-menu/files/5431829/Example.menu.files.zip)

**Comment by Gkleinereva** [comment]
Hey @denehyg - any chance you've had a chance to peek at this?  As part of my continued efforts to remedy the problem, I ran your original menu.js and my gulped menu.js through an unminifier.  I've attached the results in case you find them helpful.

Sorry to bother you, but I'd like to start working on the changes that my colleague and I are looking at.  Let me know if I can provide any additional information!  Thanks.

[og_menu_unminified.js](https://drive.google.com/file/d/1rXfuaHPe4a84ipoOtMQCWcPkW2BpP75s/view?usp=sharing)
[my_menu_unminified.js](https://drive.google.com/file/d/1T75fvtZ3NDyrLL2VzHM9CGSuNYcPRWPt/view?usp=sharing)
